packages/web/pages/api/typedefs/article.ts

```diff
@@ -0,0 +1,174 @@
+import { ApolloError } from "apollo-server-micro";
+import { authenticated } from "./scalars";
+
+export const typeDef = `
+  extend type Query {
+    article(id: ID!): Article
+    articles: [Article]
+  }
+
+  extend type Mutation {
+    createdArticle(articleInput: ArticleCreateInput!): Article
+    updatedArticle(articleInput: ArticleUpdateInput): Article
+    
+    isFavourite(articleId: ID!): Boolean
```

Comment:
Use `is` for accessing boolean, not setting. This could be `toggleFavourite` or `setFavourite`